openssl-dynamic/src/main/java/io/netty/internal/tcnative/ocsp/OcspCallback.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.internal.tcnative.ocsp;
+
+import javax.net.ssl.SSLEngine;
+
+import io.netty.internal.tcnative.SSL;
+import io.netty.internal.tcnative.SSLContext;
+
+
+/**
+ * Implementations of this interface are being called from JNI for each {@link SSLEngine} instance that
+ * has OCSP stapling enabled. Please notice that it's the same interface for client and server SSL engines
```

Comment:
nit: `SSL engines` -> `{@link SSLEngine}s` ... or just `for client and server mode`